client/layout/guided-tours/tours/editor-basics-tour.js

```diff
@@ -0,0 +1,140 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { translate } from 'i18n-calypso';
+import {
+	overEvery as and,
+} from 'lodash';
+import Gridicon from 'gridicons';
+
+/**
+ * Internal dependencies
+ */
+import {
+	makeTour,
+	Tour,
+	Step,
+	ButtonRow,
+	Next,
+	Quit,
+} from 'layout/guided-tours/config-elements';
+import {
+	isNewUser,
+} from 'state/ui/guided-tours/contexts';
+import { isDesktop } from 'lib/viewport';
+
+export const EditorBasicsTour = makeTour(
+	<Tour
+		name="editorBasicsTour"
+		version="20170321"
+		path="/post/"
+		when={ and(
+			isDesktop,
+			isNewUser,
+			) }
+		>
```

Comment:
Indentation: put `>` at the same level as `<Tour`. Same goes for the `Step` elements below.